ui/src/components/chat/compose/chatbar.rs

```diff
@@ -343,72 +378,62 @@ pub fn get_chatbar<'a>(cx: &'a Scoped<'a, super::ComposeProps>) -> Element<'a> {
                     local_typing_ch.send(TypingIndicator::Typing(*id));
                 }
             },
-            value: state
-                .read()
-                .get_active_chat()
-                .as_ref()
-                .and_then(|d| d.draft.clone())
-                .unwrap_or_default(),
+            value: state.read().get_active_chat().as_ref().and_then(|d| d.draft.clone()).unwrap_or_default(),
             onreturn: move |_| submit_fn(),
-            extensions: cx.render(rsx!(
-                // Load extensions
-                for node in ext_renders {
-                    rsx!(node)
-                }
-            )),
-            controls: cx.render(rsx!(Button {
-                icon: icons::outline::Shape::ChevronDoubleRight,
-                disabled: is_loading || disabled,
-                appearance: if *can_send.get() {
-                    Appearance::Primary
-                } else {
-                    Appearance::Secondary
-                },
-                aria_label: "send-message-button".into(),
-                onpress: move |_| submit_fn(),
-                tooltip: cx.render(rsx!(Tooltip {
-                    arrow_position: ArrowPosition::Bottom,
-                    text: get_local_text("uplink.send"),
-                })),
-            })),
+            extensions: cx.render(rsx!(for node in ext_renders { rsx!(node) })),
+            controls: cx
+    .render(
+        rsx!(
+            Button { icon : icons::outline::Shape::ChevronDoubleRight, disabled :
```

Comment:
Want to try to format this again?

Reply:
If its not causing a compile error, we can resolve it a bit later since it might be an IDE issue causing the macro to be formatted like this